src/elfcore.c

```diff
@@ -73,6 +73,8 @@ extern "C" {
     #define O_LARGEFILE 0x2000
   #elif defined(__ARM_ARCH_3__)
     #define O_LARGEFILE 0400000
+  #elif defined(__aarch64__)
+    #define O_LARGEFILE  __O_LARGEFILE
```

Comment:
I wonder if it's worthwhile to use something like, **#elif defined (__O_LARGEFILE) \n #define O_LARGEFILE __O_LARGEFILE** just to get all platforms that do define __O_LARGEFILE.

This works too.

Reply:
I think this is good idea. I will do it.